test/parallel/test-http-server-connections-checking-leak.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
```

Comment:
I'm not sure if there is a reliable way to write a unit test for this.

Reply:
Maybe we can use `onGC` to track the `Server` object.
```
const onGC = require('../common/ongc');

onGC({}, { ongc() { console.log('gc'); } });

```

Reply:
@lpinca I agree, there is no reliable way. I went with a very trivial approach. I'll probably raise the threshold a little bit to minimize false positives, but that's pretty much about it.

@theanarkh Thanks, using `onGC` was much better. Sending and updated version right now.